lib/rucio/core/transfer.py

```diff
@@ -719,10 +720,14 @@ def get_transfer_requests_and_source_replicas(total_workers=0, worker_number=0,
         # In case the source_rse and the dest_rse are connected, the list contains only the destination RSE
         # In case of non-connected, the list contains all the intermediary RSEs
         list_hops = []
+        include_multihop = core_config_get('transfers', 'use_multihop', default=False, expiration_time=600, session=session)
+        if transfertool:
```

Comment:
Needs to be checked if transfertool is `fts`, only for non-fts transfertools we want to disable this for sure.

Reply:
Since include_multihop defaults to false can't we just remove the transfertool check entirely?

Reply:
The problem is, e.g. in the ATLAS deployment it will evaluate to `True`, since we use multihop in fts. But we don't (actually can't) use multhop in GO.